.env.local.example

```
@@ -1,13 +1,5 @@
-# Google Analytics 4 Measurement ID (optional)
-# Get this from your Google Analytics 4 property
-NEXT_PUBLIC_GA_ID=G-XXXXXXXXXX
-
-# Vercel Analytics (automatically configured)
-# No additional configuration needed
-
-# Other analytics (optional)
-# NEXT_PUBLIC_CLARITY_ID=
-# NEXT_PUBLIC_HOTJAR_ID=
+## random str
+TRACKING_SALT=somethingverylongpreferably32charaactersorsomethinglikethatbutactuallydoesntreallymatteraslongitsquiterandomandnotguessableoreasilyreproducable
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)